prow/plugins/trigger/pr.go

```diff
@@ -82,6 +87,9 @@ I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that
 
 	owner := pr.Base.Repo.Owner.Login
 	name := pr.Base.Repo.Name
+	if err := ghc.AddLabel(owner, name, pr.Number, needsOkToTest); err != nil {
+		logger.Errorf("Could not add the needs-ok-to-test label to pr %d", pr.Number)
```

Comment:
Rather than logging out the error, propagate it up. I'd be happy with something like
```go
err1 := ghc.AddLabel(...)
err2 := ghc.CreateComment(...)
if err1 != nil || err2 != nil {
    return fmt.Errorf("askToJoin: error adding label: %v, error creating comment: %v")
}
return nil
```